docs/workflows/snapshotting.md

```diff
@@ -0,0 +1,37 @@
+## Snapshotting
+
+## Background
+
+Yum repositories provided by Red Hat and third parties could be updated daily or weekly.  If a user is wanting to build an image and iterate on that image, or control what patches are applied to systems they cannot use these external repositories directly.  Instead users can create point in time snapshots to create a stable baseline of content to build images upon or patch systems with.
+
+### What is snapshotting?
+
+While introspection only stores information about the repositories, snapshots store entire copies of the repositories.  This includes copies of all RPMs and metadata such as advisories.  Taking a snapshot takes much longer than introspection as all metadata is parsed and stored, and all RPMs are downloaded.
+
+The snapshots are served at a path such as  `/pulp/content/96d6d190/804288c1-13c3-40bc-9abf-ec2ed4ef4a12/8d6336b5-0569-4c3e-9341-1b3a049c67d2/`
```

Comment:
it might be helpful to add what these values are referencing (like `/pulp/content/<domain>/<distribution_path>` or something)